qt/python/mantidqt/project/workspaceloader.py

```diff
@@ -0,0 +1,27 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+
+from os import path, listdir
+from mantid.simpleapi import LoadNexusProcessed, LoadMD, Load  # noqa
```

Comment:
Can we move this line inside the `load_workspaces` method? It requires kicking up the mantid framework and we don't want that done to early.

Reply:
Makes sense and Done :+1: